lib/features/goals/pages/goal_screen.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:givt_app_kids/features/goals/cubit/goal_tracker_cubit.dart';
+import 'package:givt_app_kids/features/goals/model/family_goal.dart';
+import 'package:givt_app_kids/features/goals/widgets/family_goal_tracker.dart';
+import 'package:givt_app_kids/helpers/snack_bar_helper.dart';
+import 'package:givt_app_kids/shared/widgets/custom_progress_indicator.dart';
+
+class GoalScreen extends StatelessWidget {
+  const GoalScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocConsumer<GoalTrackerCubit, GoalTrackerState>(
+      listener: (context, state) {
+        if (state.error.isNotEmpty) {
+          SnackBarHelper.showMessage(
+            context,
+            text: state.error,
+            isError: true,
+          );
+        }
+      },
+      builder: (context, state) {
+        return Scaffold(
+          body: SafeArea(
+            child: state.currentGoal == const FamilyGoal.empty()
+                ? Center(
+                    child: Column(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        SvgPicture.asset(
+                            'assets/images/family_superheroes.svg'),
+                        const SizedBox(height: 20),
+                        Text(
+                          'Your Family Group and other\ngroups will appear here',
+                          textAlign: TextAlign.center,
+                          style: Theme.of(context).textTheme.bodyMedium,
+                        ),
+                      ],
+                    ),
+                  )
+                : Stack(children: [
+                    const FamilyGoalTracker(),
+                    if (state.currentGoal.status == FamilyGoalStatus.updating)
+                      const CustomCircularProgressIndicator()
```

Comment:
shall it be inside (part of) FamilyGoalTracker?

Reply:
If i put it inside the FamilyGoalTracker it would be overlapping the card specifically. Now it is just on screen, which I prefer, like this:
![simulator_screenshot_FC4C7ED7-BB9D-4EA9-87A8-DF5ED8387847](https://github.com/givtnl/givt-app-kids/assets/43034761/bc17fd3e-25da-4416-a03c-8a331cd2627b)